### PR TITLE
Merge final release/18.8 into develop

### PR DESCRIPTION
**Pull request by AliSoftware**
- [x] WPAndroid 18.8 submitted to Google for review
- [x] JPAndroid 18.8 submitted to Google for review

This final build also includes https://github.com/wordpress-mobile/WordPress-Android/pull/15671, which was merged yesterday and thus didn't get a chance to get into an intermediate beta before the final build.

---

Note that:
 - There was a couple of unexpected things during the release finalization, esp the About screen not being translated, and the kotlin version conflict (which printed errors in the build log but didn't make the build fail). We decided to submit anyway, See p8Qyks-28v-p2#comment-719 and p1639160990234700-slack-C02KLTL3MKM for details.
 - Multiple Release Notes translations were too long and exceeded the 500 characters limit, and were thus not included in the `release_notes.xml` files and in PlayStore submission. The Release Notes will thus fallback to English on the PlayStore for those locales.

### CODE REVIEW

File: fastlane/metadata/android/release_notes.xml [hunk 1]
```diff
@@ -1,90 +1,126 @@
...
+- Inserter menu tabs match your device's language.
+- Device media options open when creating a Gallery block.
+- Font size bug and RTL visual issues are resolved.
+- Edit Embed block link in Block Settings.
 </en-US>
-<es-ES>
-18.7:
-- Hemos ampliado el ajuste «Enlace a» para el bloque de imágenes para incluir las opciones «Archivo de medios» y «Página de adjuntos» además de la opción preexistente «URL personalizada».
-- Hemos solucionado un problema con el guardado automático del contenido, por lo que ahora se ejecuta más fluidamente.
-</es-ES>
```
Comment (AliSoftware) [inline fastlane/metadata/android/release_notes.xml:20]: `es-ES translation for release_note_188 exceeds maximum length (543).`

File: fastlane/metadata/android/release_notes.xml [hunk 2]
```diff
@@ -1,90 +1,126 @@
...
+- Tab menu penyisip sesuai dengan bahasa perangkat Anda.
+- Pilihan media perangkat terbuka saat membuat blok Galeri.
+- Masalah bug ukuran font dan tampilan RTL telah diperbaiki.
+- Mengedit tautan blok Sematan di Pengaturan Blok.
 </id>
-<it-IT>
-18.7:
-- Abbiamo ampliato l'impostazione "Collega a" per il blocco immagine per includere le opzioni "File multimediale" e "Pagina allegato" insieme all'opzione preesistente "URL personalizzato".
-- Abbiamo risolto un problema con i salvataggi automatici dei contenuti in modo che ora funzionino in modo più fluido.
-</it-IT>
```
Comment (AliSoftware) [inline fastlane/metadata/android/release_notes.xml:45]: `it-IT translation for release_note_188 exceeds maximum length (505).`

File: fastlane/metadata/android/release_notes.xml [hunk 3]
```diff
@@ -1,90 +1,126 @@
...
+- Inserter menu tabs komen overeen met de taal van je apparaat.
+- Apparaat media-opties worden geopend bij het maken van een galerijblok.
+- Lettertypegrootte bug en RTL visuele problemen zijn opgelost.
+- Bewerk de insluitblok link in Blok instellingen.
 </nl-NL>
-<pt-BR>
-18.7:
-- Ampliamos a configuração "Link para" de forma que o bloco de imagem inclua as opções "Arquivo de mídia" e "Página de anexo" juntamente com a opção já existente "URL personalizada".
-- Corrigimos um problema no salvamento automático de conteúdo para deixá-lo mais estável.
-</pt-BR>
```
Comment (AliSoftware) [inline fastlane/metadata/android/release_notes.xml:65]: `pt-BR translation for release_note_188 exceeds maximum length (546).`

File: fastlane/jetpack_metadata/android/release_notes.xml [hunk 4]
```diff
@@ -1,84 +1,107 @@
...
+- Tabs des Inserter-Menüs stimmen mit der Sprache deines Geräts überein.
+- Beim Erstellen eines Galerieblocks werden die Medienoptionen des Geräts geöffnet.
+- Problem mit der Schriftgröße und visuelle RTL-Probleme wurden behoben.
+- Bearbeite Embed-Block-Links in den Blockeinstellungen.
 </de-DE>
-<es-ES>
-18.7:
-- Hemos ampliado el ajuste "Enlace a" del bloque Imagen para incluir las opciones "Archivo de medios" y "Página de adjuntos", junto con la opción preexistente "URL personalizada".
-- Hemos resuelto un problema con el guardado automático de contenidos y ahora funciona mucho mejor.
-</es-ES>
```
Comment (AliSoftware) [inline fastlane/jetpack_metadata/android/release_notes.xml:19]: `es-ES translation for release_note_188 exceeds maximum length (588).`

File: fastlane/jetpack_metadata/android/release_notes.xml [hunk 5]
```diff
@@ -1,84 +1,107 @@
...
-<fr-FR>
-18.7 :
-- Nous avons développé le réglage « Lier à » du bloc d’image pour inclure les options « Fichier média » et « Page de fichier joint » à côté de l’option préexistante « Lien personnalisé ».
-- Nous avons résolu un problème avec les sauvegardes automatiques de contenu afin qu’elles s’exécutent désormais plus facilement.
-</fr-FR>
```
Comment (AliSoftware) [inline fastlane/jetpack_metadata/android/release_notes.xml:24]: `fr-FR translation for release_note_188 exceeds maximum length (581).`

File: fastlane/jetpack_metadata/android/release_notes.xml [hunk 6]
```diff
@@ -1,84 +1,107 @@
...
+- Tab menu penyisip sesuai dengan bahasa perangkat Anda.
+- Pilihan media perangkat terbuka saat membuat blok Galeri.
+- Masalah bug ukuran font dan tampilan RTL telah diperbaiki.
+- Mengedit tautan blok Sematan di Pengaturan Blok.
 </id>
-<it-IT>
-18.7:
-- Abbiamo ampliato l'impostazione "Collega a" per il blocco Immagine in modo da includere le opzioni "File multimediale" e "Pagina dell'allegato" accanto all'opzione preesistente "URL personalizzato".
-- Abbiamo risolto un problema con i salvataggi automatici dei contenuti per farli funzionare in modo più fluido.
-</it-IT>
```
Comment (AliSoftware) [inline fastlane/jetpack_metadata/android/release_notes.xml:39]: `it-IT translation for release_note_188 exceeds maximum length (505).`

File: fastlane/jetpack_metadata/android/release_notes.xml [hunk 7]
```diff
@@ -1,84 +1,107 @@
...
+- Menutabbladen bij de invoeger komen overeen met de taal van je apparaat.
+- Opties met apparaatmedia worden geopend wanneer je een Galerijblok aanmaakt.
+- Bug met lettertypegrootte en visuele problemen met rechts-naar-links zijn verholpen.
+- Link voor Insluitblok bewerken bij blokinstellingen.
 </nl-NL>
-<pt-BR>
-18.7:
-- Ampliamos a configuração "Link para" de forma que o bloco de imagem inclua as opções "Arquivo de mídia" e "Página de anexo" juntamente com a opção já existente "URL personalizada".
-- Corrigimos um problema no salvamento automático de conteúdo para deixá-lo mais estável.
-</pt-BR>
```
Comment (AliSoftware) [inline fastlane/jetpack_metadata/android/release_notes.xml:59]: `pt-BR translation for release_note_188 exceeds maximum length (514).`

File: fastlane/jetpack_metadata/android/release_notes.xml [hunk 8]
```diff
@@ -1,84 +1,107 @@
...
-<ru-RU>
-18.7:
-— Теперь, помимо уже имеющейся опции "Пользовательский URL-адрес", параметр "Ссылка на" для блока изображения включает опции "Медиафайл" и "Страница вложения".
-— Мы решили проблему автосохранения содержимого, и теперь операция выполняется более плавно.
-</ru-RU>
```
Comment (AliSoftware) [inline fastlane/jetpack_metadata/android/release_notes.xml:64]: `ru-RU translation for release_note_188 exceeds maximum length (502).`

**Review by AliSoftware — COMMENTED** [review]
Highlighting the Release Notes that got excluded by the tooling for being too long

**Review by jkmassel — APPROVED** [review]

**Pull request merged**